src/copyright/agent/database.cc

```diff
@@ -300,21 +300,38 @@ bool CopyrightDatabaseHandler::insertNoResultInDatabase(long int agentId, long i
 
 bool CopyrightDatabaseHandler::insertInDatabase(DatabaseEntry& entry) const
 {
-  return dbManager.execPrepared(
-    fo_dbManager_PrepareStamement(
-      dbManager.getStruct_dbManager(),
-      "insertInDatabase",
-      "INSERT INTO "
-      IDENTITY
-      "(agent_fk, pfile_fk, content, hash, type, copy_startbyte, copy_endbyte)"
-        " VALUES($1,$2,$3,md5($3),$4,$5,$6)",
-      long, long, char*, char*, int, int
-    ),
-    entry.agent_fk, entry.pfile_fk,
-    entry.content.c_str(),
-    entry.type.c_str(),
-    entry.copy_startbyte, entry.copy_endbyte
-  );
+  if(!strcmp(entry.type.c_str(),IDENTITY) || strcmp(entry.type.c_str(),"statement")){
```

Comment:
but `IDENTITY` might be equal to `"copyright"`: https://github.com/siemens/fossology/blob/a5a5276ce806f330756957b9713957f7ad3ff49d/src/copyright/agent/identity.hpp#L31

The only other option for `IDENTITY` is `"ecc"` (see `identity.hpp`). This implies that only ecc is placed in another table called ecc and everything else is placed in the table called `author`? At least this is how I understand the code.